src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripLabelTests.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.Drawing;
+
+namespace System.Windows.Forms.Tests;
+
+public class ToolStripLabelTests : IDisposable
+{
+    private readonly ToolStripLabel _toolStripLabel = new();
+    public void Dispose() => _toolStripLabel.Dispose();
+
+    [WinFormsFact]
+    public void ToolStripLabel_DefaultConstructor_SetsDefaults()
+    {
+        ToolStripLabel toolStripLabel = new();
```

Comment:
Please dispose these
```suggestion
        using ToolStripLabel toolStripLabel = new();
```